core/src/test/java/test/org/apache/spark/JavaAPISuite.java

```diff
@@ -897,6 +901,18 @@ public void persist() {
     assertEquals(1, rdd.first().intValue());
   }
 
+  @Test
```

Comment:
@tgravescs, what about disabling this test and see if it was from side effect? Logically, Python related logic does not affect Scala/Java test cases, and I presume it might be the side effect from the test it runs first.

Seems the test output is from somewhere related to matrix (e.g., `MetricsSystemSuite`); however, I am not sure where exactly it comes from.

Reply:
sure let me try this and I'll address rest of comments afterwards

Reply:
seems this was the issue. I'll have a closer look.

Reply:
its a weird way to fail but the test was hanging because of the testing config. since its local mode its not support but it was throwing an exception because testing config was on.  Simply removed the collect call because all I want to test is that java api works.